### DISCUSSION TITLE
Custom preview deployment URLs

### DISCUSSION THREAD

**Discussion by rebelchris** [vercel/community#4367] [Feature Question] [answered]
### Question

Would it be possible to assign the deployment domains (currently `{something}.vercel.app` to our own TLD as well?
I know we can do it per branch, but we'd like to see this dynamic per deployment basically.

**Comment by swarnava** [comment, marked as answer]
Hi,

You can setup Preview Deployment Suffix feature which will let you use Custom TLD for Preview deployments: https://vercel.com/docs/deployments/generated-urls#preview-deployment-suffix
